index.md

```diff
@@ -53,13 +57,13 @@ This page introduces guidelines and other standards related to web accessibility
 
 The World Wide Web Consortium (W3C) develops international Web standards: <abbr title="Hypertext Markup Language">HTML</abbr>, <abbr title="Cascading Style Sheets">CSS</abbr>, and many more. W3C's Web standards are called <dfn>W3C Recommendations</dfn>.
 
-All W3C standards are reviewed for accessibility support by the Accessible Platform Architectures ([APA](/about/groups/apawg/)) Working Group.
+All W3C standards are reviewed for accessibility support by the Accessible Platform Architectures ([APA](https://www.w3.org/WAI/about/groups/apawg/)) Working Group.
 
 The W3C standards and Working Group Notes introduced below are particularly relevant to accessibility.
 
 ## Accessibility Guidelines {#guidelines}
 
-[[Essential Components of Web Accessibility]](/fundamentals/components/) shows how web accessibility depends on several components of web development and interaction working together, and how the WAI guidelines (WCAG, ATAG, UAAG) apply.
+[Essential Components of Web Accessibility](https://www.w3.org/WAI/fundamentals/components/) shows how web accessibility depends on several components of web development and interaction working together, and how the WAI guidelines (WCAG, ATAG, UAAG) apply.
```

Comment:
This change undoes the “take the title of the target translation on translated pages” that I implemented two weeks ago, including the automatic addition of “… (in English)” that the markdown links with double brackets do. Was that change intentional?